manifests/pe_metric.pp

```diff
@@ -38,10 +39,39 @@
   }
 
   $script_file_name = "${scripts_dir}/${metric_script_file}"
+  $conversion_script_file_name = "${scripts_dir}/json2timeseriesdb"
+
+  $metrics_base_command = "${script_file_name} --metrics_type ${metrics_type} --output-dir ${metrics_output_dir}"
+
+  if !empty($metrics_server_info) {
+    $metrics_server_hostname = $metrics_server_info['hostname']
+    $metrics_server_port     = $metrics_server_info['port']
+    $metrics_server_type     = $metrics_server_info['metrics_server_type']
+    $metrics_server_db       = $metrics_server_info['db_name']
+
+    if empty($metrics_server_db) and $metrics_server_type == 'influxdb'  {
+      fail( 'When using an influxdb server you must provide the db_name to store metrics in' )
+    }
+
+    $local_metrics_command = "${metrics_base_command} | ${conversion_script_file_name} --netcat ${metrics_server_hostname} --convert-to ${metrics_server_type}"
```

Comment:
Can we make this command overridable by a parameter? It would be nice to specify this for external tools like <https://github.com/puppetlabs/puppetlabs-splunk_hec/commit/b39cd4ebda691a17d16c19a0e4f22d2db9348f71>